cmd/utils/flags.go

```diff
@@ -134,6 +134,15 @@ func printHelp(out io.Writer, templ string, data interface{}) {
 
 var (
 	// General settings
+	InitGenesisFlag = cli.StringFlag{
+		Name: "genesis",
+		Usage: "Path to the genesis json file. If the flag is not set, node will start by loading chain-data from \"data-dir\". " +
+			"Otherwise node will start by validating the genesis file, checks if genesis file is match with genesis block, and " +
+			"checks if chain configuration of the genesis file is compatible with current chain-data, apply new compatible chain " +
+			"configuration into chain-data and start the node. Otherwise node start will end up with a mis-match genesis error or an " +
+			"incompatible chain configuration error.",
```

Comment:
```suggestion
		Usage: `Path to the genesis json file, the genesis file contains
	configuration that defines the first (aka genesis) block of the chain
	and also controls how the client behaves. This flag must be provided
	the first time a node is run so that the node can initialise its
	database with the provided config. Subsequent to the first run, if the
	flag is not set, the node will load all configuration from its
	database, otherwise the node will verify that the config for the
	genesis block exactly matches the genesis block it has in its database
	and then see if the configuration that controls client behaviour is
	compatible with the configuration in the datbase. If so the client will
	update the configuration that controls client behaviour. This is the
	mechanism is used for handling forks.`,
```

I've expanded on the comment here since there is quite a lot going on with this flag. I've also used backtics to enclose the string which allows for easier formatting of the text. I also formatted it into a block as opposed to a really long line because it makes it much easier to read when it is displayed to the user.

Reply:
Note the indentation here was intentonal, it results in the displayed text being left aligned.